### PR TITLE
Spell-checked 

### PR DESCRIPTION
**Pull request by mihir-thakkar-ai**
Ran the codespell script to check for spelling errors and corrected all the errors.

### CODE REVIEW

File: docs/examples/batch-to-online.ipynb [hunk 1]
```diff
@@ -316,16 +316,16 @@
     "scaler = preprocessing.StandardScaler()\n",
     "\n",
     "for xi, yi in stream.iter_sklearn_dataset(datasets.load_breast_cancer()):\n",
-    "    xi = scaler.learn_one(xi)"
+    "    xi = scaler.fit_one(xi)"
```
Comment (gbolmier) [inline docs/examples/batch-to-online.ipynb:319]: Not an error, we replaced the `fit_` convention by `learn_` (cf. #329). Other notebooks need to be fixed.

Comment (mihir-thakkar-ai) [reply docs/examples/batch-to-online.ipynb]: > Not an error, we replaced the `fit_` convention by `learn_` (cf. #329). Other notebooks need to be fixed.

Sorry about that. I have fixed it now.

Comment (gbolmier) [reply docs/examples/batch-to-online.ipynb]: No worries @mihir-thakkar-ai, thx for your help :)

**Comment by MaxHalford** [comment]
LGTM! Thank you so much 👍

**Pull request merged**